content-cards.php

```diff
@@ -82,16 +83,22 @@ public static function init() {
 	    // shortcode preview
 		add_action( 'admin_init', 						array( 'Content_Cards', 'init_preview' ), 20 );
 		add_action( 'wp_ajax_content_cards_shortcode', 	array( 'Content_Cards', 'ajax_shortcode' ), 20 );
-
 	}
 
 	public static function uninstall() {
 		global $wpdb;
+		self::_delete_cached();
 		$q = "DELETE FROM `{$wpdb->postmeta}` WHERE `meta_key` LIKE 'content_cards_%'";
 		$wpdb->query( $q );
 		delete_option( 'content-cards_options' );
 		wp_cache_flush();
 	}
+
+	public static function deactivate() {
+		self::_delete_cached();
+		wp_cache_flush();
```

Comment:
Don't think we need to flush the object  cache as long as we use wp_delete_attachment()?